package.json

```diff
@@ -73,25 +75,21 @@
     "validator": "^6.1.0"
   },
   "devDependencies": {
-    "@silverstripe/webpack-config": "^0.2.9",
-    "babel-core": "^6.24.1",
+    "@silverstripe/webpack-config": "file:../../webpack-config",
```

Comment:
For development purposes, to replace with the proper `0.3.0` version when that is published on the `webpack-config` package